src/test/kotlin/pl/droidsonroids/gradle/jacoco/testkit/JaCoCoTestKitPluginFunctionalTest.kt

```diff
@@ -97,20 +98,33 @@ class JaCoCoTestKitPluginFunctionalTest {
 
     @Test
     fun `plugin compatible with Configuration Cache`() {
+        val testKitDir = temporaryFolder.newFolder()
+        // https://docs.gradle.org/8.1/userguide/configuration_cache.html#config_cache:not_yet_implemented:testkit_build_with_java_agent
+        temporaryFolder.root.resolve("gradle.properties").delete()
+
         temporaryFolder.newFile("build.gradle").fillFromResource("simple.gradle")
-        GradleRunner.create()
+        val initialRun = GradleRunner.create()
             .withProjectDir(temporaryFolder.root)
-            .withTestKitDir(temporaryFolder.newFolder())
+            .withTestKitDir(testKitDir)
```

Comment:
This was a bug I spotted, second run didn't really use configuration cache. I fixed my mistake + added proper assertions to ensure everything works in an expected way